server/models/User.js

```diff
@@ -20,12 +21,11 @@ const userSchema = new Schema(
       type: String,
       required: true,
     },
-    createdEvents: [
+    createdEvents: 
```

Comment:
I think we'll still want this to be an array of references to events, that way we'll be able to define routes that use the models.